src/main/java/com/configcat/UserCondition.java

```diff
@@ -0,0 +1,71 @@
+package com.configcat;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Describes a condition that is based on a User Object attribute.
+ */
+public class UserCondition implements ConditionAccessor {
+
+    /**
+     * The User Object attribute that the condition is based on. Can be "Identifier", "Email", "Country" or any custom attribute.
```

Comment:
I'd rather move these docs comments from private fields to the public accessors.